modules/apps/layout/layout-content-page-editor-web-test/src/testFunctional/ContentPageReview.testcase

```diff
@@ -80,11 +90,16 @@ definition {
 
 	@description = "This case ensures a user can delete a fragment comment after publishing a content page."
 	@priority = "5"
-	@refactorneeded
-	test DeleteComment {
-		SitePages.openPagesAdmin(siteURLKey = "test-site-name");
-
-		ContentPages.addPage(pageName = "Test Content Page Name");
+	@refactordone
+	test CanDeleteComment {
+		JSONLayout.addPublicLayout(
+			groupName = "Test Site Name",
+			layoutName = "Content Page Name",
+			type = "content");
+
+		ContentPagesNavigator.openEditContentPage(
+			pageName = "Content Page Name",
+			siteName = "Test Site Name");
 
 		PageEditor.addFragment(
```

Comment:
@liuxingye can you see if we can add fragments through JSON? Pretty sure there are existing macros here